Cargo.toml

```diff
@@ -1,7 +1,29 @@
 [package]
-name = "yago4"
-version = "0.0.0"
+name = "yago"
```

Comment:
I would prefer naming this crate yago-builder because it is only focused  on building the YAGO dumps. Most of the YAGO users should just use them and don't bother with is code.

Reply:
I thought about this as well, like this you could install it with "cargo install yago" with the semver (if there might be a yago 5 in the future written in rust) and then just use yago4/5 directly from their build dir. The Rust environment ist actually perfect for this with https://github.com/nabijaczleweli/cargo-update I run autoupdates for most of my commandline tooling directly from crates.io for example. But for sure, it's your decision. You can also yank the stuff from crates.io, I just wanted to try see how it looks like with the links and everything. I didn't even push the source code for this reason. ^^

Reply:
> Most of the YAGO users should just use them and don't bother with is code.

That's true. Personally I think it's nice to have the name reserved for future projects regarding yago. Maybe some tolling around it and the `yago` crate works like an umbrella or so. But also if I see the naming scheme of the other repository like yago3 it's not directly clear from their names that they are `yagoXY-code` for building a `yago-builder` software that builds up the knowledge database. Hmm, it would somehow need to be clear that the crate `yago-builder` builds the knowledge database version 4 even if the `yago-builder` is on SemVer 4.2 or so. Or do database versions increase with the tooling that was producing it? What is the plan for the future, will there be a yago-builder that builds yago5 as a database or is it now, that it's written in rust a kind of rolling release model and continously updated while building always the same database version as the build tool `yago-builder` itself. Because in the last case (which I had in mind) it would be absolutely making sense to keep the name short and assume, that with semver the `yago` version will be congruent with the database version from now on.